docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -118,20 +116,20 @@ NOTE: You can also edit topic properties by selecting the *Properties* tab withi
 
 [role="_additional-resources"]
 .Additional resources
-* {base-url}{getting-started-url-kafka}[_Getting started with {product-long-kafka}_^]
-* {base-url}{getting-started-rhoas-cli-url-kafka}[_Getting started with the rhoas CLI for OpenShift Streams for Apache Kafka_^]
-* {base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^]
+* {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]
+* {base-url}{getting-started-rhoas-cli-url-kafka}[Getting started with the rhoas CLI for OpenShift Streams for Apache Kafka^]
+* {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^]
 
 
 [id="ref-supported-topic-properties_{context}"]
 == Supported topic properties in {product-kafka}
```

Comment:
@jbyrne-redhat All topic properties have a table of "Property attributes" with that same heading. As opposed to "Property attributes for ___ property", which I think is excessive. And none of the tables have intro statements, again excessive.

Both of these facts are intentional given the nature of the content as reference content, like API docs etc, where we similarly have many tables or blocks with the same "Required parameters" heading or "Example request" / "Example response" etc., not specific to every single one.

Just noting in passing in light of the style focus lately. Open to any other thoughts.

Reply:
Yep, I agree with all of that. You're right that expanded captions and the addition of intro sentences would be excessive.  And I think you're right to treat reference content a little differently.